python/ribasim/tests/test_model.py

```diff
@@ -44,42 +45,48 @@ def test_invalid_node_id(basic):
     model = basic
 
     # Add entry with invalid node ID
-    model.pump.static = model.pump.static._append(
+    df = model.pump.static.df._append(
         {"flow_rate": 1, "node_id": -1, "remarks": "", "active": True},
         ignore_index=True,
     )
+    # Currently can't handle mixed NaN and None in a DataFrame
```

Comment:
Is this a pandas limitation? We could benefit from a clear guide on how we deal with / represent missing data for different data types on both Julia, Python, Arrow and GeoPackage sides.

See also [this comment](https://github.com/Deltares/Ribasim/pull/716#issuecomment-1792749731): @evetion what do you recommend for missing / default string columns; missing or ""?

Reply:
Pandas is fine, it's just that with adding new columns, mixing None (object) and NaN (float) doesn't play nice with pandera. I might've fixed that in the last commit. I will write a guide in the subsequent PR on how to handle missing values in Python. (I think it's pretty clear in Julia & geopackage).